pulp-core/requirements.txt

```diff
@@ -0,0 +1,8 @@
+# modules to install in core module
+pulpcore==3.10.0
+pulp-file==1.5.0
+pulp-container==2.3.1
+pulp-rpm==3.9.0
+pulp-deb==2.9.0
+gunicorn==20.0.4
+django-storages[boto3]==1.11.1
```

Comment:
can you run a `pip freeze` to generate this please

Reply:
Or `docker run --rm pulp-core-build /opt/pulp/bin/pip freeze --all -l -r /opt/pulp/pulp-requirements.txt > requirements.txt` :)